### PR TITLE
July release

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

**Comment by KellieHawks** [comment]
@josh-mccrowell-braze 
Don't forget to add to this page: https://www.braze.com/docs/help/release_notes

**Comment by josh-mccrowell-braze** [comment]
@KellieHawks Thanks for the reminder! Will do. This PR is just to check my math on the release notes right now - will add SDK notes, add the release notes page, etc. once those are finalized. 😸

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ 
+## Feature flags in Canvas
+Feature flags allow you to experiment and confirm your hypotheses around new features by turning them off and on for different sections of your user base. The new feature flag component in Canvas allows you to segment your audience in a Canvas based on whether a feature flag is on or off for a user. Moreover, Experiment Paths allow you optimize these conversions by testing different messages or paths against each other and determining which is most effective. See the [feature flag overview]({{site.baseurl}}/developer_guide/platform_wide/feature_flags/about/) for information about feature flags generally, or learn more about [using feature flags in Canvas]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/feature_flags/).
+
+## Managing Segments article
+The new [Managing Segments]{{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/) article covers the actions you can take to configure your segments, such as filtering a list of segments, creating segments, and editing segments.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes/2023/7_25_23.md:18]: [Managing Segments]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/)

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+page_order: 7
+noindex: true
+page_type: update
+description: "This article contains release notes for 07/25/23."
+ 
+---
+# July 25, 2023
+
+## Canvas approval 
+The new Canvas approval workflow setting adds a review process prior to launching a new Canvas. Please note that this feature is turned off by default, giving you control over its implementation. Explore more details about activating this workflow in [Canvas approvals and permissions]({{site.baseurl}}/user_guide/engagement_tools/canvas/managing_canvases/canvas_approval/).
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes/2023/7_25_23.md:12]: Can remove "Please" per the Style Guide ("Note that this feature...")

Comment (josh-mccrowell-braze) [reply _docs/_help/release_notes/2023/7_25_23.md:12]: Great callout, thanks! Will do!

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+nav_title: July 25, 2023
+page_order: 7
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2023/7_25_23.md:3]:
```suggestion
page_order: 5
```

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+noindex: true
+page_type: update
+description: "This article contains release notes for 07/25/23."
+ 
+---
+# July 25, 2023
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2023/7_25_23.md:9]:
```suggestion
# July 25, 2023 release
```

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## Managing Segments article
+The new [Managing Segments]{{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/) article covers the actions you can take to configure your segments, such as filtering a list of segments, creating segments, and editing segments.
+ 
+## Row component Content Block
+You can now save a row component as a Content Block that can then be used in all email campaigns and email messages in a Canvas. For more information on drag-and-drop Content Blocks, refer to [Content Blocks]({{site.baseurl}}/user_guide/message_building_by_channel/email/drag_and_drop/dnd_content_blocks/).
+
+## Sage AI
+Sage AI by Braze powers a collection of accessible, easy-to-use tools that lower the barriers to entry for creativity, personalization and optimization for your engagement strategy. Learn more about our Sage AI features and capabilities in our [documentation]({{site.baseurl}}/user_guide/sage_ai).
+
+## Update by phone (early access)
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2023/7_25_23.md:26]:
```suggestion
## Updating a user profile by phone number
```

File: _docs/_help/release_notes/2023/7_25_23.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Using the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track) endpoint, you can update a user profile using their phone number. This feature is currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2023/7_25_23.md:27]:
```suggestion
Using the [`/users/track`]({{site.baseurl}}/api/endpoints/user_data/post_user_track) endpoint, you can update a user profile using their phone number. 

{% alert important %}
This feature is currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
{% endalert %}
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks good - added some suggestions!

**Review by bre-fitzgerald — APPROVED** [review]
lgtm given Rachel and Kellie's comments 👍

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Left a small style comment.

**Comment by josh-mccrowell-braze** [comment]
@KellieHawks : Applied changes from review, added SDK release notes and links to changelogs, built out. I _think_ this is ready to go, but need a last close review to make sure everything is ready to ship.

File: _docs/_help/release_notes/2023.md [hunk 7]
```diff
@@ -34,4 +34,7 @@ guide_menu_list:
   - name: June 27, 2023
     link: /docs/help/release_notes/2023/6_27_23/
     fa_icon: fas fa-bug
+  - name: July 25, 2022
```
Comment (KellieHawks) [inline _docs/_help/release_notes/2023.md:37]:
```suggestion
  - name: July 25, 2023
```

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! One minor fix needed.

File: _docs/_help/release_notes/2023.md (hunk 7 above)
Comment (josh-mccrowell-braze) [reply _docs/_help/release_notes/2023.md:37]: 😭 thanks

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**